src/AlertBox/AlertBox.tsx

```diff
@@ -62,9 +62,9 @@ const TYPES = {
 };
 
 AlertBox.propTypes = {
```

Comment:
would like to see us move these to truly static fields, e.g.:
```ts
class AlertBox extends Component {
  static propTypes = {
    // ...
  };

  render() { /* ... */ }
}
```
so that our components are properly statically typed.
then we can remove the hack we have in the minimal typings.
what i'm leaning towards as part of the apps-dashboard conversion work: https://github.com/Clever/apps-dashboard/compare/tsx#diff-68c212eb090c15f35ca18fac07925830